src/airstack/onchain_graph.py

```diff
@@ -0,0 +1,759 @@
+"""
```

Comment:
comments applicable to the whole file.

- Avoid code duplication by creating generic methods
- Add error handling
- Try using configuration or env variables instead of hard coded values
- migrate graphql queries to a separate file
- Pagination handling: ensure that you don't make too many requests in a short period of time to avoid rate limiting 
- Removed unused code